public/app/plugins/datasource/prometheus/components/PromQueryField.tsx

```diff
@@ -219,6 +220,11 @@ class PromQueryField extends React.PureComponent<PromQueryFieldProps, PromQueryF
 
   onClickChooserButton = () => {
     this.setState((state) => ({ labelBrowserVisible: !state.labelBrowserVisible }));
+
+    reportInteraction('user_grafana_prometheus_editor_mode_clicked', {
```

Comment:
```suggestion
    reportInteraction('user_grafana_prometheus_metrics_browser_clicked', {
```

Reply:
I think we should use some other label. Or am I confused?

Reply:
Yeah no that was a copy/paste error 🤦